CalibPPS/ESProducers/python/ctppsLHCInfo_cff.py

```diff
@@ -2,6 +2,8 @@
 
 # by default, LHCInfo is now loaded from CondDB using a GT
 ctppsLHCInfoLabel = cms.string("")
+ctppsLHCInfoPerLSLabel = cms.string("")
+ctppsLHCInfoPerFillLabel = cms.string("")
```

Comment:
why are these needed? the `fillDesc` makes sure there is a default of `""` so none of these should be needed

Reply:
addressed in commit https://github.com/cms-sw/cmssw/pull/42515/commits/fd20a63bde3d3fcaa1deb60e6fc1b267ad413ec7, it will be squashed after testing and addressing comments